test/db/formats/pdb

```diff
@@ -104,6 +109,7 @@ CMDS=!rz-bin -P ${RZ_FILE} | grep -ao _RTC_ErrorNumber
 EXPECT=<<EOF
 _RTC_ErrorNumber
```

Comment:
Why this is broken? Since it's enum it's quite well supported, shouldn't be a problem

Reply:
This is caused by the RBTree problem.

Reply:
Were you able to solve this problem?